### ISSUE TITLE
Gradient intensity

### ISSUE THREAD

**Issue by tyrann** [timozattol/wallpaper-generator#7]
It would be nice to be able to provide a gradient intensity. On my system, the per-polygon gradient is usually not strong enough so that the lattice looks more like a plain gradient on the whole screen.

**Comment by timozattol** [comment]
I think the "gradient intensity" is only affected by the number of polygons and the distance between the two chosen colors.

The problem we have is - I think - that in the palettes I provided, some colors are really close to each other. I thought of a solution, which would be to use "bipartite palettes", which are composed of two sets: a set of light colors and a set of dark colors. Then we would sample one color from each set, which will always result in a nice gradient.

**Comment by tyrann** [comment]
That would be a good solution to the problem, could we possibly randomize the color generation process and force 1 random dark color to be matched with 1 random light color?

**Comment by timozattol** [comment]
Yes that's a nice idea! :+1: I was also getting tired of the palettes I'm using right now ;)
